src/main/java/io/vlingo/common/completes/CFCompletes.java

```diff
@@ -751,6 +751,16 @@ public String toString() {
       };
     }
 
+    private <O> Function<T, CompletableFuture<O>> composableFunction(final Function<T, O> userFunction) {
+      return (T t) -> {
+        O outcome = userFunction.apply(t);
+        if (outcome instanceof CFCompletes) {
+          return (CompletableFuture<O>) ((CFCompletes<?>) outcome).state().future;
```

Comment:
Unfortunately, I'm relying on a specific implementation of Completes here. I haven't found a better way to do this (yet).

Reply:
@jakzal What do you need that's different? Does the Completes interface API need to change? Feel free to make changes, hopefully non-breaking ones.

Reply:
Is it possible we'd be dealing with two different Completes implementations here? i.e. CFCompletes and SomeCustomCompletes.

Reply:
Nope. `BlahBlahCompletes<T>`, where `BlahBlah` is currently `CF`, is it.